src/components/layout.css

```
@@ -0,0 +1,30 @@
+code,
+pre {
+    font-family: Fira Code,SFMono-Regular,Consolas,Liberation Mono,Menlo,monospace;
+}
+
+pre {
+    overflow: auto;
+    background-color: #f6f8fa;
+    border-radius: 3px;
+    line-height: 1.45;
+    padding: 16px;
+}
+
+code {
+    padding: .2em .4em;
+    margin: 0;
+    font-size: 85%;
```

percent가 em과 비슷한 효과를 줄 수 있긴함니다

```suggestion
    font-size: 0.85em;
```